package.json

```diff
@@ -32,7 +32,7 @@
     "eslint-plugin-prettier": "^3.1.1",
     "file-loader": "^5.0.2",
     "prettier": "^1.19.1",
-    "three": "^0.111.0",
+    "three": "^0.128.0",
```

Comment:
Not new in this PR but technically the peerDependancy should also really point at this same version I would think, since it technically wont work with older versions... Really not a big deal though.